### PR TITLE
woodpecker-*: 1.0.0 -> 1.0.1

### PR DESCRIPTION
**Pull request by adamcstephens**
## Description of changes

https://github.com/woodpecker-ci/woodpecker/releases/tag/v1.0.1

- SECURITY
  - Fix WOODPECKER_GRPC_VERIFY being ignored 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/continuous-integration/woodpecker/woodpecker-package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "vite",
-    "build": "vite build",
+    "build": "vite build --base=/BASE_PATH",
```
Comment (ambroisie) [inline pkgs/development/tools/continuous-integration/woodpecker/woodpecker-package.json:11]: I foresee an issue here, I'll try building this at some point today.

Comment (adamcstephens) [reply pkgs/development/tools/continuous-integration/woodpecker/woodpecker-package.json:11]: I was able to build and update my server without issues.

**Comment by adamcstephens** [comment]
Result of `nixpkgs-review pr 248069` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>woodpecker-agent</li>
    <li>woodpecker-cli</li>
    <li>woodpecker-server</li>
  </ul>
</details>

-----

Result of `nixpkgs-review pr 248069` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>woodpecker-agent</li>
    <li>woodpecker-cli</li>
    <li>woodpecker-server</li>
  </ul>
</details>

-----

Result of `nixpkgs-review pr 248069` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>woodpecker-agent</li>
    <li>woodpecker-cli</li>
    <li>woodpecker-server</li>
  </ul>
</details>

-----

Result of `nixpkgs-review pr 248069` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>woodpecker-agent</li>
    <li>woodpecker-cli</li>
    <li>woodpecker-server</li>
  </ul>
</details>

File: pkgs/development/tools/continuous-integration/woodpecker/woodpecker-package.json (hunk 1 above)
Comment (ambroisie) [reply pkgs/development/tools/continuous-integration/woodpecker/woodpecker-package.json:11]: Ah I had misunderstood what that option meant (I thought it did something on the file-system). Looking at the PR that introduced it, I see that it's part of their support for sub-folder proxying.

**Review by ambroisie — APPROVED** [review]

**Pull request merged**